packages/components/ng-ovh-payment-method/src/payment-method-helper.service.js

```diff
@@ -1,11 +1,17 @@
 import { get } from 'lodash-es';
 
-import { IBAN_BIC_RULES } from './payment-method.constants';
+import {
+  IBAN_BIC_RULES,
+  TYPE_INTEGRATION_ENUM,
+} from './payment-method.constants';
+
+import AdyenService from './components/integration/component/adyen/service';
 
 export default class OvhPaymentMethodHelperService {
   /* @ngInject */
   constructor($translate) {
     this.$translate = $translate;
+    this.AdyenService = AdyenService;
```

Comment:
Required ? I mean using `this.AdyenService` or `AdyenService` looks the same, and not used in html.

Reply:
You're right, no need to have it in `this`